master/buildbot/interfaces.py

```diff
@@ -33,10 +35,19 @@ class BuilderInUseError(Exception):
     pass
 
 
-class WorkerTooOldError(Exception):
+class WorkerSetupError(Exception):
     pass
 
 
+WorkerTooOldError = WorkerSetupError
+deprecatedModuleAttribute(
+    Version("buildbot", 2, 8, 5),
```

Comment:
```suggestion
    Version("buildbot", 2, 9, 0),
```

We only deprecate APIs on minor version bumps.